tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "allowJs": true,
+    "declaration": false,
```

Comment:
Why?

Reply:
I believe this was just so that we could re-use the hydra.js file without converting it to TS but it should all just be replaced with the auto-generated client and revert this piece

Reply:
Ah nice, I agree! - thank you for the review by the way :)